scss/mixins/_grid.scss

```diff
@@ -29,7 +29,7 @@
 @mixin make-col($size: false, $columns: $grid-columns) {
   @if $size {
     flex: 0 0 auto;
-    width: divide(100%, divide($columns, $size));
+    width: percentage(divide($size, $columns));
```

Comment:
I guess this has been done on purpose 🤔

Reply:
oops sorry missed your comment before I hit the merge button

Reply:
Yeah, I had to revert it as were started getting random percentages on our widths (e.g., instead of 75%, 75.000000019% or so). Going with this for now with the shortened precision.